client/app/visualizations/chart/plotly/utils.js

```diff
@@ -36,6 +37,10 @@ export const ColorPalette = Object.assign({}, BaseColors, {
 
 const ColorPaletteArray = values(BaseColors);
 
+function cleanNumber(value) {
+  return isUndefined(value) ? value : (plotlyCleanNumber(value) || 0.0);
```

Comment:
What if value is not supposed to be a number? I think it will be safer to return `value` if `plotlyCleanNumber` return `undefined` (`BADNUM`).

Also why not have it as part of `normalizeValue` call?

Does the tooltip even take normalized values?

Reply:
Generally, all y-values (y, y-error, size) should be numbers; it means that actually we don't need to use `normalizeValue` there (noticed it after your comment), but just `cleanNumber`; in opposite, x-values may be any (date/time, category, number), so we should sanitize them with `normalizeValue`, but not with `cleanNumber` (it's not important for x-values because we dont do any computing with them - just use as-is; but we use y-values to implement stacking (line/area chars) and percent normalization, so it **should** be a number).

Reply:
@arikfr Updated the PR: there is the only chart type where `y` may be non-number - it's `bubble` chart; for other types non-numeric `y` does not make any sense.